index.html

```diff
@@ -56,5 +56,141 @@
+                    <h1 class="p1">Data Dashboard<br>Healthcare</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="languages">
+                        <a href="" class="button">html</a>
+                        <a href="" class="button">bootstrap</a>
+                        <a href="" class="button">Ruby</a>
+                    </div>
+                </div>
+            </div>
+            <div class="work work--4-3">
+                <div class="container">
+                    <h1 class="p1">Website Portfolio</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="languages">
+                        <a href="" class="button">html</a>
+                        <a href="" class="button">bootstrap</a>
+                        <a href="" class="button">Ruby</a>
+                    </div>
+                </div>
+            </div>
+            <div class="work work--4-4">
+                <div class="container">
+                    <h1 class="p1">Profesional Art Printing Data</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="languages">
+                        <a href="" class="button">html</a>
+                        <a href="" class="button">bootstrap</a>
+                        <a href="" class="button">Ruby</a>
+                    </div>
+                </div>
+            </div>
+            <div class="work work--4-5">
+                <div class="container">
+                    <h1 class="p1">Data Dashboard<br>Healthcare</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="languages">
+                        <a href="" class="button">html</a>
+                        <a href="" class="button">bootstrap</a>
+                        <a href="" class="button">Ruby</a>
+                    </div>
+                </div>
+            </div>
+            <div class="work work--4-6">
+                <div class="container">
+                    <h1 class="p1">Website Portfolio</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="languages">
+                        <a href="" class="button">html</a>
+                        <a href="" class="button">bootstrap</a>
+                        <a href="" class="button">Ruby</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about-me" id="about">
```

Comment:
- [ ] Good work you have done on your styling 👍 . I can see that there is so much margin-top specified for this section. I think you should consider adjusting the margins for this section to match the margin in the [Figma design template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/176787830-948886ad-8359-4d32-b6dc-f5e08f2e22de.png)  | ![image](https://user-images.githubusercontent.com/63473584/176787882-35c1117d-d629-4b55-81c4-c70429b98606.png)  |